Catch_ve_tests/Catch_ve_tests.cpp

```diff
@@ -2,6 +2,7 @@
 #pragma warning(disable : 4127)
 
 #define DCON_TRAP_INVALID_STORE
+#define __AVX512BW__
```

Comment:
You wouldn't want to define this here: setting the target instruction set should define it. Otherwise, you guarantee that these tests will fail on older systems